content/soda-gettingstarted/installation-using-ansible.md

```diff
@@ -110,48 +69,86 @@ By default the `host_ip` is set to `127.0.0.1` i.e. localhost.
 
-To install Delfin, update the file `group_vars/delfin.yml` and change the value of `enable_delfin` to `true`
+- Enable following configurations  
+  - In file installer/ansible/group_vars/delfin.yml `enable_delfin: true`  
+  - In file installer/ansible/group_vars/srm-toolchain.yml `install_srm_toolchain: true`  
+  - In file installer/ansible/group_vars/dashboard.yml `enable_dashboard: true`  
+  - **[Click here](#configure-delfin-installation) for other supported configuration details**
+---
 
-```bash
-# Install delfin (true/false)
-enable_delfin: true
-```
+
+#### Enable Terra  
+ Terra installs SODA On Premise only.  
+- Enable following configurations  
+  - In file installer/ansible/hotpot.yml  update the value  `enable_hotpot : true`.
+  - In file installer/ansible/group_vars/dashboard.yml `enable_dashboard: true`
+  - In file installer/ansible/group_vars/common.yml `host_ip : <User's IP address, eg. 127.0.0.1>`  
+  - **[Click here](#configure-soda-on-premise-installation) for other supported configuration details**   
+
+#### Enable Strato
+Strato installs SODA Multicloud only.
+- Enable following configurations  
+  - In file installer/ansible/group_vars/gelato.yml  update the value  `enable_gelato : true`.
+  - In file installer/ansible/group_vars/dashboard.yml `enable_dashboard: true`
+  - In file installer/ansible/group_vars/common.yml `host_ip : <User's IP address, eg. 127.0.0.1>`  
+  - **[Click here](#enable-storage-service-plans-in-multicloud) other supported configuration details**
 
 ---
+### Run SODA Installer
+Run SODA installer ansible playbook to start the deployment
 
-#### Enable SRM Toolchain installation (optional)  
-Delfin produces metrics which can be consumed by any of the exporters that are supported. Currently Delfin supports the Prometheus and Kafka exporters.
-The SRM Toolchain is required to view the metrics and visualization in the SODA Dashboard.  
-Update the file `ansible/group_vars/srm-toolchain.yml` and change the value of `install_srm_toolchain` to `true`.  
-If this value is set to false then the metrics and visualization will not be available using SODA Dashboard. 
+Check if the hosts can be reached
 
+```bash  
+sudo -E env "PATH=$PATH" ansible all -m ping -i local.hosts
 ```
-install_srm_toolchain: true
+
+```bash
+sudo -E env "PATH=$PATH" ansible-playbook site.yml -i local.hosts
+# You can use the -vvv or -vv option to enable verbose display and debug mode.
+[verbosity level: -vv < -vvv]
+sudo -E env "PATH=$PATH" ansible-playbook site.yml -i local.hosts -vvv
 ```
----
 
-#### Enable Storage Service Plans in multi-cloud (optional)
+SODA ansible installer supports installation using tags. To install a particular service use the playbook as follows
+```bash
+# This installs only delfin
+sudo -E env "PATH=$PATH" ansible-playbook site.yml -i local.hosts -vvv --tags delfin
```

Comment:
Please check if the `delfin` tag works. @joseph-v Does it work?

Reply:
> Please check if the `delfin` tag works. @joseph-v Does it work?

Yes